catalog/plugin/catalog-plugin-defaultsecurityattributevalues/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -15,21 +15,28 @@
 <blueprint xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
            xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
 
-    <bean id="securityAttributes" class="org.codice.alliance.catalog.core.api.impl.types.SecurityAttributes"/>
+    <bean id="securityAttributes"
+          class="org.codice.alliance.catalog.core.api.impl.types.SecurityAttributes"/>
 
     <bean id="defaultMarkingsPlugin"
           class="org.codice.alliance.catalog.plugin.defaultsecurity.DefaultSecurityAttributeValuesPlugin">
-        <cm:managed-properties persistent-id="org.codice.alliance.catalog.plugin.defaultsecurity.DefaultSecurityAttributeValuesPlugin"
-                               update-strategy="container-managed"/>
+        <cm:managed-properties
+                persistent-id="org.codice.alliance.catalog.plugin.defaultsecurity.DefaultSecurityAttributeValuesPlugin"
+                update-strategy="component-managed" update-method="update"/>
         <argument ref="securityAttributes"/>
-        <property name="classification" value="classification"/>
-        <property name="releasability" value="CountryOfAffiliation"/>
-        <property name="codewords" value="FineAccessControls"/>
-        <property name="disseminationControls" value="disseminationControls"/>
-        <property name="otherDisseminationControls" value="otherDisseminationControls"/>
-        <property name="ownerProducer" value="ownerProducer" />
+        <argument>
+            <map>
+                <entry key="classification" value="classification"/>
+                <entry key="releasability" value="CountryOfAffiliation"/>
+                <entry key="codewords" value="FineAccessControls"/>
+                <entry key="disseminationControls" value="disseminationControls"/>
+                <entry key="otherDisseminationControls" value="otherDisseminationControls"/>
+                <entry key="ownerProducer" value="ownerProducer"/>
+            </map>
+        </argument>
     </bean>
 
-    <service ranking="99" ref="defaultMarkingsPlugin" interface="ddf.catalog.plugin.PreIngestPlugin"/>
+    <service ranking="99" ref="defaultMarkingsPlugin"
```

Comment:
I would just add a comment here that the ranking is here so the plugin occurs last during processing (if that still is the case).  Otherwise, this might bite someone down the road when they develop a PreIngestPlugin that they want firing last (and can't figure out why it's not).